package.json

```diff
@@ -112,9 +112,11 @@
     "archiver": "^2.1.0",
     "balena-config-json": "^2.0.0",
     "balena-device-init": "^5.0.0",
+    "balena-device-status": "^3.1.2",
     "balena-image-manager": "^6.0.0",
     "balena-preload": "^8.2.0",
     "balena-sdk": "^11.17.0",
+    "balena-semver": "2.0.2",
```

Comment:
I'm using a pinned version b/c later versions require a typescript upgrade to work, which I tried but ended up complicating this PR a lot.